docs/content/how-to/aws/create-infra-iam-separately.md

```diff
@@ -134,7 +135,118 @@ where
     to your workers. An ssh key will have been generated for you and stored as a secret in the 
     same namespace as the hosted cluster.
 
-Running this command should result in the following resources getting applied to your cluster:
+# Working with yaml
+## Creating the AWS infra & IAM
+
+Use the `hypershift create infra aws` command:
+
+    hypershift create infra aws --name CLUSTER_NAME \
```

Comment:
How about other flags like `--additional-tags`, `--enable-proxy`, `--zones`?

Reply:
1. Implemented additional-tags and zones.
2. We can add proxy later if it makes sense. You can still use the CLI for the proxy.
3. Tags are set once on the HCI and then those tags a re-used on IAM & cluster create.